src/controllers/userController.js

```diff
@@ -202,9 +204,91 @@ userController.post('/reset-password', async (req, res) => {
     }
 });
 
+//Approving/ Unapproving user
```

Comment:
Space after "//" and before the next "/"

Reply:
fixed